src/taxonomy/data/api.js

```diff
@@ -48,3 +52,12 @@ export async function deleteTaxonomy(pk) {
 export function getTaxonomyExportFile(pk, format) {
   window.location.href = getExportTaxonomyApiUrl(pk, format);
 }
+
+/**
+ * Downloads the template file for import taxonomies
+ * @param {('json'|'csv')} format
+ * @returns {void}
+ */
+export function getTaxonomyTemplateFile(format) {
+  window.location.href = getTaxonomyTemplateApiUrl(format);
+}
```

Comment:
Instead of doing this, I think you can just add this URL as a link direct using the download property. 

i.e. 

`<a href={getTaxonomyTemplateApiUrl(format)} download=`taxonomy-template.${format}`>`

Reply:
Done! I skipped the `download` attribute: the download file name is set in the backend API.